src/index.ts

```diff
@@ -3,6 +3,8 @@ import senecaSn from './seneca-sn'
 import * as Seneca from 'seneca-await'
 import {startDispatch} from './dispatch'
 import cfg from './cfg'
+import {firebase} from './process-firebase'
```

Comment:
That is actually what this is doing. The imported function is just "how I create firebase roots from environment variables" That fb root is then injected into the plugins, i.e. startDispatch and startMetrics.

The reason I moved it to a separate file is that I can see running the metrics program as a separate process very easily, and I thought it might be good to standardize to make it easier to spin up new scripts and processes.